Assets/Scripts/IsoSpriteSorting.cs

```diff
@@ -67,6 +71,7 @@ public void SetupStaticCache()
 
     private void RefreshBounds()
     {
+
         cachedBounds = new Bounds2D(renderersToSort[0].bounds);
```

Comment:
Add to this a `UseSortingGroup` check, to use sorting groups without selecting `SpriteRenderer` (like the Particle System or Spine Animation mesh). That fixes the sorting sorting error, but a full fix requires a change in `IsoSpriteSorting.CalculateBoundsIntersection()` to compare the objects without bounds with other (return true)
```suggestion
        if (UseSortingGroup)
            cachedBounds = new Bounds2D(renderersToSort[0].bounds);
```